src/python/workshop/utilities.py

```diff
@@ -52,15 +50,13 @@ async def get_file(self, project_client: AIProjectClient, file_id: str, attachme
                 file.write(chunk)
 
         self.log_msg_green(f"File saved to {file_path}")
-        # Cleanup the remote file
-        await project_client.agents.delete_file(file_id)
 
     async def get_files(self, message: ThreadMessage, project_client: AIProjectClient) -> None:
         """Get the image files from the message and kickoff download."""
         if message.image_contents:
             for index, image in enumerate(message.image_contents, start=0):
                 attachment_name = (
-                    "unknown" if not message.file_path_annotations else message.file_path_annotations[index].text + ".png"
```

Comment:
This is needed as occasionally the CI wont return the file with the file extension and then it creates a bad user experience for the attendee

Reply:
reverted this change back